components/Frame/Popover/UserNav.js

```diff
@@ -199,8 +198,10 @@ const UserNav = ({
                       )}
                       <NavLink
                         {...fontStyles.sansSerifLight16}
-                        route='pledge'
-                        params={{ package: 'DONATE' }}
+                        href={{
+                          pathname: '/angebote',
+                          query: { group: 'DONATE' }
```

Comment:
```suggestion
                          query: { package: 'DONATE' }
```

There is no group "DONATE", but a package.